packages/vue/__tests__/e2e/hydration-strat-idle.html

```diff
@@ -30,9 +30,6 @@
         setTimeout(() => {
           console.log('resolve')
           resolve(Comp)
-          requestIdleCallback(() => {
```

Comment:
It’s no longer needed because the `loader` will also execute in `requestIdleCallback`.